public/app/types/plugins.ts

```diff
@@ -17,3 +17,35 @@ export interface PluginMetaInfo {
     small: string;
   };
 }
+
+export interface PluginInfo {
+  author: {
+    name: string;
+    url: string;
+  };
+  description: string;
+  links: string[];
+  logos: { small: string; large: string };
+  screenshots: string;
+  updated: string;
+  version: string;
+}
+
+export interface Plugin {
```

Comment:
maybe PluginListItem is more clear name here

Reply:
It would be nice yes, but it will cause duplicate imports with the PluginListItem component. Not really sure what to call it. Suggestions?